### ISSUE TITLE
What should the WG interview template spreadsheet look like

### ISSUE THREAD

**Issue by daiClegg** [dbrownukk/EFD_v2#26]
Need a definitive target for generation

Should it have the characteristic resource 'expected' values ?
if so should they be for overwrite or as cross check additional values?

**Comment by daiClegg** [comment]
Still to be finally confirmed and clarified but i think there are 3 spreadsheets:

A generic blank:

[Dai_WGSpreadsheet_Blank.xlsx](https://github.com/dbrownukk/EFD_v2/files/1620641/Dai_WGSpreadsheet_Blank.xlsx)

A customised template for a specific WG interview:

[ExampleBlankCommunityInterview.xlsx](https://github.com/dbrownukk/EFD_v2/files/1620643/ExampleBlankCommunityInterview.xlsx)

And a completed WG interview spreadsheet:

[1_Hosea_Nakisunga_ Middle.xlsx](https://github.com/dbrownukk/EFD_v2/files/1620644/1_Hosea_Nakisunga_.Middle.xlsx)

Outstanding questions for clarification are:

- Are the expected values for Characteristic Resources not required in the template for a specific WG interview?

- Are the additional details in the completed interview optional extras not to be loaded?

**Comment by daiClegg** [comment]
The expected values for Characteristic Resources are not required in the generated template.

**Comment by daiClegg** [comment]
Still chasing up the answer about additional market data collected in 1_Hosea_Nakisunga_Middle.xlsx.

Let's not wait on that. 
If it is required i'll raise a new issue for that, but outside current scope.

**Comment by dbrownukk** [comment]
Closing, I suspect this will reappear but closing for now..
